### PR TITLE
Default to `--pretty-print=never` for `-e` and stdin input

### PR DESCRIPTION
**Pull request by 0xAdk**
see #237

Pretty sure this is a breaking change since I added a parameter to `InterpreterResult::to_markup()`

### CODE REVIEW

**Comment by sharkdp** [comment]
Thank you very much for your contribution!

This looks great. I just merged #97. It would be great if you could rebase your branch. Let me know if you need help resolving the conflicts.

**Comment by sharkdp** [comment]
> Pretty sure this is a breaking change since I added a parameter to `InterpreterResult::to_markup()`

I don't think the `numbat` crate has any users (yet). I also didn't really design it as a proper "public" library so far. But thank you for taking that into consideration!

**Comment by 0xAdk** [comment]
ok it's rebased, I also went ahead and changed https://github.com/sharkdp/numbat/pull/240/files#diff-723b19ccffb6afcf74a982f6dd78c112191d462e58d360c9fa49167157b1d278R136 if you don't mind.

File: numbat-cli/src/main.rs [hunk 1]
```diff
@@ -133,27 +133,12 @@ impl Cli {
...
-        } else {
-            config.main.load_prelude
-        };
-        config.main.load_user_init = if args.no_prelude || args.no_init {
-            false
-        } else {
-            config.main.load_user_init
-        };
+        config.main.load_prelude &= !args.no_prelude;
+        config.main.load_user_init &= !(args.no_prelude || args.no_init);
```
Comment (sharkdp) [inline numbat-cli/src/main.rs:137]: Thank you. I knew there was a better way to write this, I was just too lazy to think yesterday :smile:

**Comment by sharkdp** [comment]
Thank you very much!

**Pull request merged**